modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/UserNotificationTypeTest.java

```diff
@@ -86,6 +91,54 @@ public void tearDown() throws PortalException {
 			user2.getUserId());
 	}
 
+	@Test
+	public void testIsDelivery() throws Exception {
+		NotificationTemplate notificationTemplate =
+			notificationTemplateLocalService.addNotificationTemplate(
+				_createNotificationContext(
+					Arrays.asList(
+						NotificationRecipientSettingUtil.
+							createNotificationRecipientSetting(
+								"userScreenName", user1.getScreenName()),
+						NotificationRecipientSettingUtil.
+							createNotificationRecipientSetting(
+								"userScreenName", user2.getScreenName())),
+					NotificationRecipientConstants.TYPE_USER));
+
+		_userNotificationDeliveryLocalService.addUserNotificationDelivery(
+			user1.getUserId(), childObjectDefinition.getPortletId(),
+			_classNameLocalService.getClassNameId(
+				childObjectDefinition.getClassName()),
+			UserNotificationDefinition.NOTIFICATION_TYPE_UPDATE_ENTRY,
+			UserNotificationDeliveryConstants.TYPE_WEBSITE, false);
+
+		_userNotificationDeliveryLocalService.addUserNotificationDelivery(
+			user2.getUserId(), childObjectDefinition.getPortletId(),
+			_classNameLocalService.getClassNameId(
+				childObjectDefinition.getClassName()),
+			UserNotificationDefinition.NOTIFICATION_TYPE_UPDATE_ENTRY,
+			UserNotificationDeliveryConstants.TYPE_WEBSITE, true);
+
+		_executeNotificationObjectAction(
+			notificationTemplate, childObjectDefinition, group.getGroupKey(),
+			user1);
+
+		_executeNotificationObjectAction(
+			notificationTemplate, childObjectDefinition, group.getGroupKey(),
+			user2);
+
+		List<NotificationQueueEntry> notificationQueueEntries =
+			notificationQueueEntryLocalService.getNotificationQueueEntries(
+				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
```

Comment:
@mariuo use this [method](http://github.com/liferay/liferay-portal/blob/65e821b7fc45365d39ca7c39c2fae49309b0f20b/modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/UserNotificationTypeTest.java#L362-L390) to assert the the notification queue entries and define the expected user full name as `user2.getFullName()`